www/js/app.js

```diff
@@ -328,57 +328,60 @@ $('#prefix').on('keydown', function (e) {
     }
 });
 // Search for titles as user types characters
-$('#prefix').on('keyup', function (e) {
+const prefixElement = document.getElementById('prefix')
```

Comment:
This line should be moved up to around 279, so that you can use `prefixElement` also there instead of `document.getElementById('prefix')` that appears on line 280.